Makefile

```diff
@@ -265,38 +265,5 @@ benchmark_fast_check:
 benchmark_gradle_test:
 	gradle-profiler --benchmark --scenario-file gradle/performance.scenarios gradleTest
 
-## Gradle cache node
-GRADLE_CACHE_NODE_TAG=9.9
-
-# publish to internal repo to avoid rate limits problems
-internal_publish_gradle_cache_node_image:
```

Comment:
Why do we remove it?
Have we made a final decision about Enterprise?

Reply:
Nope, we can remove it later, yes